starters/rest-api/src/models/users.rs

```diff
@@ -2,6 +2,7 @@ use async_trait::async_trait;
 use chrono::offset::Local;
 use loco_rs::{auth::jwt, hash, prelude::*};
 use serde::{Deserialize, Serialize};
+use serde_json::Map;
```

Comment:
Starters folder are deprecated and move the `loco-new`. 
please revert all the changes under this path